Export Analysis Reports/Sdl.Community.ExportAnalysisReports/Service/StudioService.cs

```diff
@@ -0,0 +1,244 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using Sdl.Community.ExportAnalysisReports.Helpers;
+using Sdl.Community.ExportAnalysisReports.Interfaces;
+using Sdl.Community.ExportAnalysisReports.Model;
+using Sdl.Community.Toolkit.Core.Services;
+using Sdl.ProjectAutomation.Core;
+using Sdl.ProjectAutomation.FileBased;
+
+namespace Sdl.Community.ExportAnalysisReports.Service
+{
+	public class StudioService : IStudioService
```

Comment:
As before, there are a lot of "if inversion" recommendations in this class too.

Reply:
Indeed, the places where the inversions could be applied were updated and the changes will be reflected within the next commit